blocks/header/header.js

```diff
@@ -212,27 +219,43 @@ export default async function decorate(block) {
     decorateIcons(nav);
 
     // Add search form
-    const searchIcon = navSearch.children[0].children[0].children[0];
+    const searchIcon = navSearch.querySelector('.icon');
```

Comment:
This is less performant.

Reply:
after a few tests, it's not conclusive because it tasks a similar amount of time for both solutions
![Screenshot 2023-03-15 at 08 54 42](https://user-images.githubusercontent.com/117984204/225243390-80bfe5ac-8303-45dd-a720-46790325e458.png)
![Screenshot 2023-03-15 at 08 56 55](https://user-images.githubusercontent.com/117984204/225243746-462ba0e8-706a-4563-b87b-f91930610489.png)
![Screenshot 2023-03-15 at 08 57 32](https://user-images.githubusercontent.com/117984204/225243977-1d258ca7-ac8d-4aed-93a7-d72d4487bac4.png)
in summary, the change is more clear and easier to understand what element is saved in the variable